pg_search/src/bootstrap/create_bm25.rs

```diff
@@ -398,6 +398,18 @@ fn create_bm25_impl(
         $func$ LANGUAGE plpgsql"
     ))?;
 
+    let schema_oid_query = format!(
+        "SELECT oid FROM pg_namespace WHERE nspname = {}",
+        spi::quote_literal(index_name)
+    );
+    let schema_oid = Spi::get_one::<pg_sys::Oid>(&schema_oid_query)
```

Comment:
I believe the index is always created in the same schema as the table. So you could just use the schema oid of the `PgRelation` above.

Reply:
Yes, that's correct

Reply:
Hmm I'm finding that if I just use the `pg_relation.namespace_oid()` (without a separate `SPI` lookup), I'm breaking these two scenarios:

1. Index is dropped in the same transaction as being created.
2. A partial index column in dropped.

I don't have a great intuition for exactly why these break... but the extra `SPI` lookup seems to fix it. Seeing as the cost of the lookup is one-time and negligible I'm inclined to keep it, but it would be nice to grok why that fix works. @eeeebbbbrrrr any guess on this one?

Reply:
Can you define "breaking"?  Is Postgres raising an ERROR or what?

Reply:
For scenario 1 in the `uuid` test I'm getting:
```
ERROR:  could not open relation with OID 1241901
```

For scenario 2 in the `bm25_partial_index_alter_and_drop` test I'm getting:
```
 ERROR:  cannot DROP TABLE "test_partial_index" because it is being used by active queries in this session
```

Reply:
Can you push this code?

Reply:
Sure, I just pushed and the latest commit is causing the test failures.